apis/postgresql/v1alpha1/connectionsecret.go

```diff
@@ -0,0 +1,12 @@
+package v1alpha1
+
+// ConnectableInstance is the composable type for enabling connection secret.
+type ConnectableInstance struct {
```

Comment:
Why so many wraps around this string?

Reply:
I made this type composable by intention, to make this type reusable for other API specs in the future, for example `PostgresqlReplicated`. That way we won't have to duplicate `WriteConnectionSecretToRef`.